README.md

```diff
@@ -110,124 +71,91 @@ After you're finished please remove all the comments and instructions!
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 > Add a link to your deployed project.
+### Deployment
+
+- I will deploy the project with [render](https://render.com/docs/deploy-rails), after which you can access the live version of the project in the Live Demo section. I/We tried it several times following the process explained but it failed.The link for failed deployment is attached :(https://dashboard.render.com/web/srv-cfrqlv4gqg46pjo8eg6g)
+:(https://budget-ek.onrender.com)
```

Comment:
- [ ] Hello @Doheera-kosi I understand that despite all the ways you tried deploying on render, It kept failing. I have been struggling with deploying rails app too but I have been trying different available options. You know it's among the project requirements to have a deployed version. That's why I am going to provide all other resources and platforms you can try so that you can deploy your project successfully 😃 Don't worry this time it won't be difficult
There is Railway: Here is the tutorial that will help you -- [Youtube](https://www.youtube.com/watch?v=x1cyzFD736M&t=22s)  | [article](https://dev.to/timnans/deploy-rails-application-to-railwayapp-12gl)
There is also Fly.io: [YouTube](https://www.youtube.com/watch?v=6Zp9y8nF5rE&t=124s) | [Article](https://fly.io/docs/rails/getting-started/) 

Please try out each one and I am sure that you will find one which succeeds, just follow the procedure 😎